website/content/docs/auth/azure.mdx

```diff
@@ -390,3 +395,73 @@ namespace Examples
 </CodeBlockConfig>
 
 </CodeTabs>
+  
+## Authentication Flows and Role Restrictions
+  
+This section provides more details about the authentication flows and associated restrictions that be supplied in the the role definition.
+  
+There are 3 core flows and associated methods of restricting a role in the Azure Auth Method:
+- JWT Flow: Identity based restrictions that use claims from the JWT.
+- VM Flow: Virtual Machine (VM) based restrictions that use data from the Virtual Machine.
```

Comment:
```suggestion
- VM Flow: Virtual Machine (VM) based restrictions that use data from the virtual machine.
```

Reply:
A little tricky when it comes whether or not to capitalize virtual machines. 

1. If used as a noun, don't capitalize.
2. If it's part of the name of a feature, or precedes something else, for instance, Virtual Machine Scale Set, then we'd want to capitalize it.